src/pages/settings/bank-accounts/common/queries.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { endpoint } from 'common/helpers';
+import { request } from 'common/helpers/request';
+import { useQuery } from 'react-query';
+import { route } from 'common/helpers/route';
+
+export function useBankAccountsQuery(params: { id: string | undefined }) {
+  return useQuery(
+    route('/api/v1/bank_integrations/:id', { id: params.id }),
+    () =>
+      request(
+        'GET',
+        endpoint('/api/v1/bank_integrations/:id', { id: params.id })
+      ),
+    { staleTime: Infinity }
+  );
+}
```

Comment:
You can expand this by providing generic interface for the response, so you get full end to end types. 

Here's example: https://github.com/invoiceninja/ui/blob/main/src/pages/recurring-invoices/common/queries.ts#L23

Reply:
> You can expand this by providing generic interface for the response, so you get full end to end types.
> 
> Here's example: https://github.com/invoiceninja/ui/blob/main/src/pages/recurring-invoices/common/queries.ts#L23

done